susemanager-utils/testing/docker/scripts/reportdb_schema_idempotency_test_pgsql.py

```diff
@@ -274,8 +274,27 @@ def get_stripped_lines(file):
                 for line in re.sub("\t", "       ", fd.read()).splitlines()
             ]
 
-    file_initial = get_stripped_lines(initial_dump)
-    file_migrated = get_stripped_lines(migrated_dump)
+    def get_sorted_tables(lines):
+        in_table = False
+        res = []
+        table = []
+        for line in lines:
+            if not in_table:
+                res.append(line)
+                if line.endswith("FROM stdin;"):
+                    in_table = True
+            else:
+                if line == "\.":
```

Comment:
Could you paste an example of a dump? I don't really understand the code without it (not saying the fix is wrong :-))

Reply:
BTW, I tried to relaunch `java_pgsql_tests` but it seems gitarro is not launching it even if the checkbox is marked. The original test results linked at the statuses seem to be gone.

Reply:
As no java code changed, it does not run this check. No idea why it did 1 time

Reply:
Thanks!